create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/ScopusConverter.java

```diff
@@ -262,6 +262,21 @@ private UnsupportedCitationTypeException getUnsupportedCitationTypeException() {
             .map(CitationTypeTp::getCode);
     }
 
+    private JournalArticle generateJournalArticle(JournalArticleContentType contentType) {
+        JournalArticle journalArticle = generateJournalArticle();
+        journalArticle.setContentType(contentType);
+        return journalArticle;
+    }
+
+    private JournalArticle generateJournalArticle() {
+        JournalArticle journalArticle = new JournalArticle();
+        extractPages().ifPresent(journalArticle::setPages);
+        extractVolume().ifPresent(journalArticle::setVolume);
+        extractIssue().ifPresent(journalArticle::setIssue);
+        extractArticleNumber().ifPresent(journalArticle::setArticleNumber);
```

Comment:
Trengs det ikke tester for dette?

Reply:
Ligger test(er) i en annen branch, jeg skal legge inn det vi har etterhvert